components/nimbus/src/glean_metrics.rs

```diff
@@ -0,0 +1,160 @@
+// -*- mode: Rust -*-
```

Comment:
Do we really need to check this in? I already hate glean because it always changes metrics.md line endings for no good reason, and having this checked in is almost certainly going to make me hate it even more. We manage to avoid the uniffi generated rust, so I'd strongly prefer to avoid glean generated rust.

Reply:
No, I added it to this draft simply to make it easier to review and see. I would agree that generated code shouldn't be added and once I get the particulars of how best to get the glean_parser step integrated into the build system I'll be sure and untrack it.